docs/concepts.rst

```diff
@@ -1,128 +1,130 @@
+Learn more about customizing CumulusCI automation in the [TODO: link Customization section].
 
-Orgs
-^^^^
+force-app (or src)
+^^^^^^^^^^^^^^^^^^
+
+The main body of the project's code and metadata lives in the default package directory, commonly shown as the ``force-app`` folder for Salesforce DX-format projects and the ``src`` folder for Metadata API-format projects. **force-app** defines what will be included when you release a managed package from your CumulusCI project (or when you release an unlocked package, or if you're not releasing the package at all but running the ``deploy`` task to get the metadata into an org in unmanaged form).
+
+orgs directory
+^^^^^^^^^^^^^^
 
-The ``.json`` files found in the ``orgs`` directory define the Salesforce DX org configurations that are available to the project. We cover scratch org management in depth in TODO: link Scratch Org Management. 
+The ``.json`` files found in the ``orgs`` directory define the Salesforce DX org configurations that are available to the project. We cover scratch org management in depth in [TODO: link Scratch Org Management].
 
-Datasets
+datasets
 ^^^^^^^^
 
-Each project may have one or more *datasets*: on-disk representations of record data that can be inserted into Salesforce orgs, and which can also be modified and re-captured during the evolution of the project. Datasets are stored in the ``datasets`` directory. Learn more about datasets in TODO: link Automating Data Operations.
+Each project may have one or more **datasets**: on-disk representations of record data that can be inserted into Salesforce orgs, and which can also be modified and re-captured during the evolution of the project. Datasets are stored in the ``datasets`` directory. Learn more about datasets in [TODO: link Automating Data Operations].
 
-Robot Framework Browser Automation
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+robot
+^^^^^
 
-CumulusCI integrates Robot Framework browser automation for end-to-end testing. Each project contains a ``robot`` directory, which stores the resources and test suites for the project's Robot Framework test suites. New CumulusCI projects start with a simple Robot test case that creates a Contact record. 
+Robot Framework provides browser automation for end-to-end testing. Each project contains a ``robot`` directory, which stores the project's Robot Framework test suites. New CumulusCI projects start with a simple Robot test case that creates a Contact record.
 
-While Robot Framework is used primarily for browser automation testing, in a small number of situations, Robot Framework may also be harnessed to help configure orgs where other strategies and APIs are insufficient.
+While Robot Framework is used primarily for browser automation testing, it can also be harnessed to help configure orgs where other strategies and APIs are insufficient. [TODO LINK TO EXPLAIN THIS FURTHER?]
```

Comment:
```suggestion
While Robot Framework is primarily for browser automation testing, it's also useful for configuring orgs when other strategies and APIs are insufficient. [TODO LINK TO EXPLAIN THIS FURTHER?]
```
Is it more common to say "automated browser testing" than "browser automation testing"? I didn't suggest the change because I'm not sure.

Reply:
@davidmreed, @davisagli, @Br4nd0R: Thoughts?

Reply:
I think "automated browser testing" is clearer.